projects/core/src/occ/adapters/product/converters/occ-product-search-page-normalizer.ts

```diff
@@ -20,11 +29,38 @@ export class OccProductSearchPageNormalizer
       ...target,
       ...(source as any),
     };
+    this.normalizeFacetValues(target);
     if (source.products) {
       target.products = source.products.map(product =>
         this.converterService.convert(product, PRODUCT_NORMALIZER)
       );
     }
     return target;
   }
+
+  /**
+   *
+   * In case there are so-called `topValues` given for the facet values,
+   * we replace the facet values by the topValues, simple because the
```

Comment:
Is this a typo: `simple` instead of `simply`?

Reply:
typo ;)